### ISSUE TITLE
Store patch assumes relationships are kind of hasMany and has links

### ISSUE THREAD

**Issue by matthewvalimaki** [pangratz/ember-data-meta-links-improvements#3]
I am getting `You tried to get the 'testCase' relationship on a 'execution' via record.hasMany('testCase'), but the relationship is of type 'belongsTo'. Use record.belongsTo('testCase') instead` with
```
...,
relationships: {
   "test-case": {
      id: 5,
      type: "test-case"
   }
}
```
and this seems to be caused by https://github.com/pangratz/ember-data-meta-links-improvements/blob/b80e15bc470fb03a384374bf38972a737b0357dc/addon/patches/store.js#L18. If I comment out this line error goes away and things work. It actually does two assumptions:
1) there are `links` (which it doesn't)
2) it is `hasMany` and not `belongsTo`

**Comment by pangratz** [comment]
Yep, as pointed out in the README this is currently not implemented 😔 .

**Comment by matthewvalimaki** [comment]
@pangratz oops! Apologies for now spotting that! The README does clearly state under "links for relationship references" that "belongs to" is not yet implemented. I'll try to figure out a PR for it as I already looked into it.

**Comment by matthewvalimaki** [comment]
I am still going to work on this. Had to work on something else first.

**Comment by matthewvalimaki** [comment]
Similar issue with DELETE which causes `didSaveRecord` to be called and it makes a reference to `data.meta`, which with response 204 does not exist. I'll work on this as well.

UPDATE: forgot that I already logged #2 for this.

**Comment by matthewvalimaki** [comment]
@pangratz hey do you have time to take a look at PR #5? What I would like to do is start completing the other missing features of this addon, keep it up-to-date etc. I would like to avoid doing a brand new release of it.
